modules/subscriptions.php

```diff
@@ -1064,6 +1108,13 @@ function jetpack_do_subscription_form( $instance ) {
 	}
 	$instance['show_subscribers_total'] = empty( $instance['show_subscribers_total'] ) ? false : true;
 
+	if (
+		array_key_exists( 'subscribe_text', $instance ) &&
+		is_array( $instance['subscribe_text'] )
+	) {
+		$instance['subscribe_text'] = $instance['subscribe_text'][0];
```

Comment:
If it's not numeric array, this could be an issue?  Should this just be an `array_shift()` instead?